frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -220,6 +220,16 @@ if Device:isAndroid() then
         end,
     }
 
+    -- camera key events
+    common_settings.android_camera_keys = {
+        text = _("Camera key toggles touchscreen"),
+        checked_func = function() return G_reader_settings:isTrue("camera_key_toggles_touchscreen") end,
+        callback = function()
+            local current = G_reader_settings:isTrue("camera_key_toggles_touchscreen")
+            G_reader_settings:saveSetting("camera_key_toggles_touchscreen", not current)
```

Comment:
`G_reader_settings:flipNilOrFalse("camera_key_toggles_touchscreen")` makes that a single line.